recipes/udunits2/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $(uname) != Darwin ]]; then
+    yum install -y automake libtool texinfo
```

Comment:
> Not according to manylinux.

I have the feeling that `manylinux` does not seem to be addressing build tools. It looked to me that the discussion was focused on libraries and dependencies. (I might be wrong as I haven't read much about it yet.)

> In practice, that might mean we want a `automake` distribution which we can mark as a build dep of udunits2.

I will take a look into that.